packages/api-server/api_server/routes/tasks/scheduled_tasks.py

```diff
@@ -234,7 +239,8 @@ async def update_schedule_task(
 
         async with tortoise.transactions.in_transaction():
             if except_date:
-                event_date_str = except_date.isoformat()
+                event_date_str = convert_date_server_timezone_iso_str(except_date)
+                logger.info(f"Updating event with iso format date: {event_date_str}")
```

Comment:
Include the subject (task id).

Reply:
Thanks for catching that, that'll be useful. I've also added more logs in the route function
https://github.com/open-rmf/rmf-web/pull/902/commits/66db89323ac657088d4118dbbeba626fe35a1ae6

We can move logs to debug during the next phase of the project when these features stabilize